src/ui/public/vis/response_handlers/legacy.js

```diff
@@ -18,26 +18,21 @@
  */
 
 import _ from 'lodash';
-import { tabifyAggResponse } from '../../agg_response/tabify';
 import AggConfigResult from '../../vis/agg_config_result';
 import { VisResponseHandlersRegistryProvider } from '../../registry/vis_response_handlers';
 
+
 const LegacyResponseHandlerProvider = function () {
 
   return {
     name: 'legacy',
-    handler: function (vis, response) {
+    handler: function (table) {
```

Comment:
response handlers accept only tabified data as an argument

Reply:
You should also fix the typing in `response_handler.d.ts` to remove the `vis` param from it.

Reply:
its already removed right ?